pygsti/modelmembers/states/tpstate.py

```diff
@@ -11,16 +11,25 @@
 #***************************************************************************************************
 
 
-import numpy as _np
+from __future__ import annotations
+from typing import Tuple, TYPE_CHECKING
+if TYPE_CHECKING:
+    import torch as _torch
```

Comment:
Could this raise ImportError that would not be caught by the block below and be a problem?

Reply:
Nope! It looks weird, but ``TYPE_CHECKING`` is a constant that's equal to ``False`` at runtime.